3-networks/modules/vpn-ha/README.md

```diff
@@ -0,0 +1,23 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
We should add an overview of what the module does here, even if it is only a couple of sentences.